### ISSUE TITLE
[filterscripts] FSS filter geometry (Trac #2102)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2094]
Read the FSS filter geometry (IC79 or IC86) from the G frame, instead of as a configuration option.  This involves making the `polygonCutStringNumbers` choice inside the module, instead of in the tray segment.

Migrated from https://code.icecube.wisc.edu/ticket/2102
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:15:13", 
    "description": "Read the FSS filter geometry (IC79 or IC86) from the G frame, instead of as a configuration option.  This involves making the `polygonCutStringNumbers` choice inside the module, instead of in the tray segment.", 
    "reporter": "david.schultz", 
    "cc": "joertlin, blaufuss", 
    "resolution": "fixed", 
    "_ts": "1550067313248429", 
    "component": "combo reconstruction", 
    "summary": "[filterscripts] FSS filter geometry", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2017-10-17T16:54:23", 
    "milestone": "", 
    "owner": "jbraun", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20180607T19:52:10: olivas changed milestone from "December 2017" to "Summer Solstice 2018"

**Comment by icecube-trac** [comment]
Trac update at 20180626T22:58:19: 
* jbraun changed owner from "" to "jbraun"
* jbraun changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20180626T22:58:32: 
* jbraun changed status from "assigned" to "closed"
* jbraun changed resolution from "" to "fixed"
* jbraun commented:

> In changeset 163736:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="163736"
> Read the FSS filter geometry (IC79 or IC86) from the G frame.  Closes #2094.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:15:13: 
* olivas changed milestone from "Summer Solstice 2018" to ""
* olivas commented:

> Milestone Summer Solstice 2018 deleted
